CONTRIBUTING.md

```diff
@@ -6,47 +6,62 @@ We love pull requests from everyone. By participating in this project, you agree
 to abide by the [code of conduct].
 
 [code of conduct]: CODE_OF_CONDUCT.md
+## How to contribute
+1. Fork the repo.
```

Comment:
With markdown you can create numbered lists with just prepending each line with a `1. {your text}`.  When its rendered it will render it with number bullets.  I find this to be a best practice, because you can just move lines around and not worry about renumbering the lines correctly.